### PR TITLE
Create Role Modal. 

### PR DESCRIPTION
**Pull request by Prajwal7842**
Logout and login error is fixed.

### CODE REVIEW

File: client/src/components/AppNavbar.js [hunk 1]
```diff
@@ -28,10 +29,13 @@ class AppNavbar extends Component {
...
   };
-
+  componentDidMount(){
+    const { isAuthenticated, user } = this.props.auth;
+    console.log("here"+isAuthenticated);
+  }
   render() {
     const { isAuthenticated, user } = this.props.auth;
-
+    console.log("het"+isAuthenticated);
```
Comment (rohitkatlaa) [inline client/src/components/AppNavbar.js]: remove it

File: client/src/actions/authActions.js [hunk 2]
```diff
@@ -9,21 +9,32 @@ import {
...
+
 } from './types';
 
 // Check token & load user
 export const loadUser = () => (dispatch, getState) => {
   // User loading
   dispatch({ type: USER_LOADING });
-
+  const token = localStorage.getItem("token");
+  console.log("TOKEN " + token);
```
Comment (rohitkatlaa) [inline client/src/actions/authActions.js]: remove it

**Comment by rohitkatlaa** [comment]
@Prajwal7842 Remove all the console logs

**Comment by Prajwal7842** [comment]
Forgot to remove console.log statements. Will send a new one with clean code.

**Pull request closed without merging**